### PR TITLE
Pull integrations integration test fixes to 2.x

### PR DESCRIPTION
**Pull request by Swiddis**
### Description
Looks like fixes were lost in backporting, this PR copies them over. This fixes the integration tests action for integrations on the 2.x branch.

### Issues Resolved
N/A

### Check List
- [X] New functionality includes testing.
  - [X] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by RyanL1997 — APPROVED** [review]
Hi @Swiddis, thanks for taking this on. This is LGTM, btw since @mengweieric enabled the lint, I think we should also fix some of the lint errors for the test files.

File: .eslintrc.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ module.exports = {
     'cypress/assertion-before-screenshot': 'warn',
     'cypress/no-force': 'warn',
     'cypress/no-async-tests': 'error',
+    // default no-unused-vars doesn't handle when vars are only used in types, use extended version
+    // See: https://typescript-eslint.io/rules/no-unused-vars
+    'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': 'error'
```
Comment (ps48) [inline .eslintrc.js:50]: @mengweieric Can you please review this change in es lint?

Comment (joshuali925) [reply .eslintrc.js:50]: also add this to ignore `_*` variables that should be unused? https://github.com/joshuali925/observability-langchain/blob/ebdd5152a0d5cdeddea56f2e93ecc882b045833e/.eslintrc.js#L27-L31
```js
    '@typescript-eslint/no-unused-vars': [
      'error',
      {
        argsIgnorePattern: '^_',
        varsIgnorePattern: '^_',
        caughtErrorsIgnorePattern: '^_',
      },
    ],
```

i remember eric had a PR for this

Comment (ps48) [reply .eslintrc.js:50]: @Swiddis Can you please undo the lint changes and rebase from the latest 2.x?

Comment (Swiddis) [reply .eslintrc.js]: Applied the changes. I notice that in the docs from the comment they explicitly disable the old unused-vars lint, but we keep both enabled. I wonder if that difference matters? The config from Eric's PR at least seems to not report errors on the file I was fixing here so it probably isn't important for now.

Comment (joshuali925) [reply .eslintrc.js]: it's contained by recommended

**Review by ps48 — APPROVED** [review]

**Review by mengweieric — APPROVED** [review]

**Pull request merged**